core/domain/user_services.py

```diff
@@ -146,6 +155,13 @@ def validate(self):
             raise utils.ValidationError(
                 'Invalid email address: %s' % self.email)
 
+        if not isinstance(self.role, basestring):
+            raise utils.ValidationError(
+                'Expected role to be a string, received %s' % self.role)
+        if self.role not in role_services.PARENT_ROLES:
+            raise utils.ValidationError('Role %s does not exist.' % self.role)
+
```

Comment:
Remove extra newline. We only have two newlines above top-level function/class definitions.